######################################################
# Message for optimization based on GPS for Rtab-Map 
######################################################

Header header

int32 loopClosureId

geometry_msgs/Pose gps_pose
